### DISCUSSION TITLE
Add a custom branch for CS 1.6 Servers

### DISCUSSION THREAD

**Discussion by Nightcr4wler** [GameServerManagers/LinuxGSM#4622] [Ideas]
Hi, i recently has installed a CS 1.6 dedicated server through Linux GSM, and after a install the HLDS came with version from 25th anniversary (9907), then this version not works with ReHLDS, só i've used steam_legacy branch to roll back to pre-25th anniversary build (8684) and installed the ReHLDS manually, thats bring more stability and resources for the servers, so the idea is create a branch to install the ReHLDS and ReGameDLL to bring more options for users.

**Comment by anzz1** [comment]
You can use [rehldsupdatetool](https://github.com/anzz1/rehldsupdatetool) for easy way to update ReHLDS
